Library/Homebrew/utils/github/api.rb

```diff
@@ -222,7 +235,18 @@ def self.open_rest(
 
         args += ["--dump-header", T.must(headers_tmpfile.path)]
 
-        output, errors, status = curl_output("--location", url.to_s, *args, secrets: [token])
+        curl_options = {
+          print_stdout:    print_stdout,
+          print_stderr:    print_stderr,
```

Comment:
`print_stdout` is always `false` by default, and `print_stderr` is always set to `false` by `curl_output`, so we don't need to pass these two.

Reply:
Thanks, great catch @reitermarkus!